src/app/pages/give/give.js

```diff
@@ -202,6 +214,12 @@ export default class Give extends CMSPageController {
         if (invalid) {
             event.preventDefault();
             this.update();
+        } else {
+            localStorage.setItem('donation:first_name', this.el.querySelector('[name=First_Name]').value);
```

Comment:
First_Name is from the form at the top of give.html where the user enters the information. The input type name (first_name) is in the hidden form I am submitting to salesforce. I did not use the same names so the query would find the correct fields (First_Name vs first_name).